node/src/container_chain_spawner.rs

```diff
@@ -363,6 +453,10 @@ impl ContainerChainSpawner {
                 );
             }
         }
+
+        if state.assigned_para_id.unwrap() != container_chain_para_id && state.next_assigned_para_id.unwrap() != container_chain_para_id{
+            delete_container_chain_db(&self.container_chain_cli.base_path);
```

Comment:
Is this path the correct one? I didn't find any other that could be more explicit to point to the db in the current scope.